src/components/Tooltip/Tooltip.vue

```diff
@@ -19,6 +20,12 @@
 import 'floating-vue/dist/style.css';
 import Icon from '@/components/Icon/Icon.vue';
 
+import DOMPurify from 'dompurify';
```

Comment:
I think we can just import the directive here?
```suggestion
import {sanitizeHtml} from '@/directives/stripUnsafeHtml';
```